cypress/integration/switch.spec.js

```diff
@@ -0,0 +1,43 @@
+describe('switch v1', () => {
+  it('works', () => {
+    cy.visit('/#/doodad/Switch')
```

Comment:
That's true, but if you test everything at once, then you produce a lot of extra noise that doesn't help you fix the violations.

In this test, I'm specifically targeting the switch components when I perform the axe scans. If you instead just scan the whole page, then you end up getting a false positive on a violation because the header of the page is wrong (not relevant to the ui component under test).  

That said, we could setup the same looping with Cypress if you want to to do that way.

Reply:
Ahh one factor here is that I want noredink-ui's styleguide to be an example of doing things right. To that end, it's important to me that the whoolee example be accessible. And also, I want employees with disabilities to be able to use internal tooling conveniently. So I don't think the failures are noise -- just less vital than the failures within the component itself